Sources/SourceControl/Repository.swift

```diff
@@ -32,18 +33,18 @@ public struct RepositorySpecifier: Hashable, Sendable {
     }
 
     /// The location of the repository as string.
-    public var url: String {
+    public var url: CanonicalPackageURL {
         switch self.location {
-        case .path(let path): return path.pathString
-        case .url(let url): return url.absoluteString
+        case .path(let path): return CanonicalPackageURL(path.pathString)
+        case .url(let url): return CanonicalPackageURL(url.absoluteString)
```

Comment:
Why this change? The canonical URL isn't necessarily the actual URL 🤔

Reply:
I removed this, it wasn't necessary.